### ISSUE TITLE
Truncate additional meshlet buffers

### ISSUE THREAD

**Issue by JMS55** [gwihlidal/meshopt-rs#61]
After calling `meshopt_buildMeshlets()`, in addition to truncating the meshlet buffer, we should also truncate the vertex and triangle buffers like so:

```rust
let last = meshlets[count - 1];
meshlet_verts.truncate((last.vertex_offset + last.vertex_count) as usize);
meshlet_tris.truncate((last.triangle_offset + ((last.triangle_count * 3 + 3) & !3)) as usize);
meshlets.truncate(count);
```

**Comment by Uriopass** [comment]
I believe this was done in https://github.com/gwihlidal/meshopt-rs/commit/2727315f96c20855a617d015c304d29a20fb996c

**Comment by JMS55** [comment]
Ah yeah, I'll just wait for a new release then.

**Comment by Uriopass** [comment]
released 0.4.1 as I believe this is no breaking change
